### DISCUSSION TITLE
All incomming/outgoing emails end up in "junk" folder

### DISCUSSION THREAD

**Discussion by kamarya** [emersion/maddy#425] [Q&A]
I have setup two domains lets say **A.com** and **B.com**. I have setup the mail server on **A.com** and it works fine (for whaterver is _@A.com_). For domain **B.com** I have set MX records to **mx.A.com** so A.com provides email services for both. For **B.com** I have made `https://mta-sts.B.com/.well-known/mta-sts.txt` with the following content:
```
mode: enforce
max_age: 604800
mx: mx.A.com
```

Why all incoming/outgoing emails to/from **B.com** end up in junk folder ? What am I missing ?

P.S. I use Mozilla Thunderbird as client

**Comment by foxcpp** [comment]
Can you check the header of messages sent to Junk and post the contents of Authorization-Results field? This may be caused by SPF/DKIM mismatch.

**Comment by kamarya** [comment]
I do not see anything failing in the header.
```
Authentication-Results: dkim=pass spf=pass
```

**Reply by foxcpp** [reply]
Then it must be something about recipient spam policy. You can try marking emails from B.com as "non-spam" in its UI, this might help.
